tests/fixtures.py

```diff
@@ -1,106 +1,88 @@
 import hpotk
 import pytest
 
+from genophenocorr.data import ProteinTestMetadataService
 from genophenocorr.model import *
 from genophenocorr.model.genome import GRCh38, GenomicRegion, Region, Strand
 
-
 def make_region(contig: str, start: int, end: int) -> GenomicRegion:
     return GenomicRegion(GRCh38.contig_by_name(contig), start, end, Strand.POSITIVE)
 
+@pytest.fixture
```

Comment:
I think we can set `@pytest.fixture(scope='session')` here. A fixture with `scope='session'` is created only once and used for _all_ tests, as opposed to creating a fixture for each test.

A small detail, but perhaps worth knowing about.